### ISSUE TITLE
Web.config registration obsolete now?

### ISSUE THREAD

**Issue by sunstatevallaycruiser** [blackcity/Backload#117]
We've updated a 2.1 release to the latest 2.2.7 release. It seems that we do not use to register the component in Web.config anymore. Is this correct?

**Comment by Auersberg** [comment]
Right. Before 2.2.7 (2.2.2.7) we had a dependency on System.Configuration where we leveraged the ASP.NET configuration system to read Backload's configuration at application startup. 
Because ASP.NET Core does not have this functionality and we want to share as much code as possible with traditional .NET, we removed dependency in Core and full .NET. Now we read/parse the config file with a standard XmlSerializer. As a side effect this is faster and more reliable.
